glide.lock

```diff
@@ -49,8 +57,12 @@ imports:
   - gf256
   - qr
   - qr/coding
+- name: github.com/matttproud/golang_protobuf_extensions
+  version: 3247c84500bff8d9fb6d579d800f20b3e091582c
+  subpackages:
+  - pbutil
 - name: github.com/miekg/dns
-  version: 58f52c57ce9df13460ac68200cef30a008b9c468
+  version: a8a1273f08d171f75254d2157ee546ea65bd0a27
```

Comment:
For these packages that changed (github.com/miekg/dns, github.com/tylerb/graceful, and github.com/xenolf/lego), are you upgrading them or is this accidentally reverting when Chris updated them last week?